bc_obps/registration/api/user_operator.py

```diff
@@ -42,6 +43,14 @@ def is_approved_admin_user_operator(request, user_guid: str):
     return 200, {"approved": approved_user_operator}
 
 
+@router.get("/user-operator-id/{user_guid}", response={200: UserOperatorIdOut, codes_4xx: Message})
+def user_operator_id(request, user_guid: str):
+    user_operator = get_object_or_404(
+        UserOperator, user_id=user_guid, role=UserOperator.Roles.ADMIN, status=UserOperator.Statuses.APPROVED
```

Comment:
I'm not sure we should filter on admin roles. Even if reporters can't edit any of the forms, I think we might need their operator id to know what to show them